collaboration-service/controller/express-controller.js

```diff
@@ -0,0 +1,31 @@
+import { ormDeleteRoom as _deleteRoom } from '../model/collaboration-room-orm.js'
+import { deleteRoom as _redisDeleteRoom } from '../model/redis-repository.js'
+import { COLLAB_ROOM_PREFIX } from '../constants.js'
+
+export const deleteRoom = async (req, res) => {
+  const { roomId } = req.params
+  if (!roomId) {
+    return res.status(400).json({ message: 'Room ID not provided.' })
+  }
+
+  const collabRoomId = `${COLLAB_ROOM_PREFIX}${roomId}`
+  const [dbResult, redisResult] = await Promise.allSettled([
+    _deleteRoom(collabRoomId),
+    _redisDeleteRoom(collabRoomId), //redis operation doesn't fail or throw erors
```

Comment:
I know I previously said that we probably don't need to clear the cache, but it turns out we do.

 If user A and user B are in a room owned by user A, and user B deletes his/her account, the room in the DB is cleared. However, if user A finds another match and creates a room, collab-svc will still read from Redis since the room is owned by user A. 

Therefore we need to clear the cache.